src/main/java/io/levchugov/petproject/handler/strategy/Strategy.java

```diff
@@ -0,0 +1,13 @@
+package io.levchugov.petproject.handler.strategy;
+
+import org.telegram.telegrambots.meta.api.methods.PartialBotApiMethod;
+import org.telegram.telegrambots.meta.api.objects.CallbackQuery;
+
+import java.io.Serializable;
+
+public interface Strategy {
```

Comment:
May be this name represents a logic of interface better? 
```suggestion
public interface CallBackProcessStrategy {
```